public/index.html

```diff
@@ -14,6 +14,15 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>IncluCivics</title>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-74QEPWXNZS"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'G-74QEPWXNZS');
```

Comment:
If this is your personal account, have you recorded it in our accounts, Airtable, @wduffee?

Reply:
@combinatorist yep!